### DISCUSSION TITLE
WTA anyone manage to play sound over RSTP for Tapo C200/C200?

### DISCUSSION THREAD

**Discussion by niaosuan** [blakeblackshear/frigate#12423] [Ask A Question]
### What is your question:

I wish to play some recorded voice for camera at my house door, want to ask If anyone manage to play sound over RSTP for Tapo C200/C200?

**Comment by hawkeye217** [comment]
A quick search in the discussion forum came up with this: https://github.com/blakeblackshear/frigate/discussions/8567
